doc/whats_new/v1.1.rst

```diff
@@ -340,6 +340,14 @@ Changelog
   to train and pickle the model on 64 bit machine and load the model on a 32
   bit machine for prediction. :pr:`21552` by :user:`Loïc Estève <lesteve>`.
 
+:mod:`sklearn.multioutput`
+...................
+- |Enhancement| Added method `decision_function` to 
+  :class:`multioutput.MultiOutputClassifier` which allows to use the
+  `decision_function` of the provided estimator in the multi-class multi-label
```

Comment:
I think you mean "multioutput multiclass classification" (several targets, each target has a variable number of classes). But I am not sure if and how `decision_function` should  also handle the "multilabel classification" case (several targets, each target is a binary variable).

We might need to record the value of [type_of_target(y_train)](https://scikit-learn.org/stable/modules/generated/sklearn.utils.multiclass.type_of_target.html#sklearn.utils.multiclass.type_of_target) on the estimator to be able to predict a datastructure of the right type.